### DISCUSSION TITLE
No prompt for password

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#2564] [Unsupported Versions]
When setting up a standalone deployment, I notice after going through installation the password wasn't required to get into sudoers mode, which is weird. Do you know why this might be the case?

**Comment by jertel** [comment]
More information would be helpful. Was this an ISO-based install, or network? Was it on your own machine, or on a cloud instance, such as EC2? Have you looked at the `/etc/sudoers` file to begin troubleshooting the behavior?

**Reply by kennysden** [reply]
Sorry didn't realize you responded. But this was an iso based install, after running through setup as usually and the networking piece after that reboot, I noticed it stop asking for the password. This VM use an EXSi hypervisor.

No I haven't looked in the /etc/sudoers yet. But I can take a look in there.

**Reply by jertel** [reply]
Yes, look at `/etc/sudoers`, towards the bottom of that file. If you see a line with the `onion` user, and a `NOPASSWD` parameter then that will explain the behavior. Let me know what you find.
